cm-mlops/script/get-ml-model-resnet50-tvm/_cm.json

```diff
@@ -25,6 +25,7 @@
   ],
   "tags": [
     "get",
+    "raw",
```

Comment:
Actually we use "raw" tag to distinguish between tvm models (or any other optimized ones) and the original ones by putting "raw" tag only to the original ones. All other changes are good to be approved. Thanks for them.